test/e2e/e2e_suite_test.go

```diff
@@ -49,6 +49,8 @@ var _ = AfterSuite(func() {
 		if e2e.VeleroInstalled {
 			err := testdata.UnsinstallVelero()
 			Expect(err).NotTo(HaveOccurred())
+			_, err = utils.Aws("s3", "rm", "s3://tce-velero-e2e-test-backup", "--recursive")
```

Comment:
this could give surprises to other e2e runs expecting files to be present (say while doing restore)

I think we should create a unique dir under this bucket for each e2e run and delete only that directory while doing cleanup